lib/asciidoctor-pdf/theme_loader.rb

```diff
@@ -86,8 +86,10 @@ def load hash, theme_data = nil
   private
 
   def process_entry key, val, data
-    if key == "admonition_icons"
-      data[key] = symbolize_keys val
+    if key.start_with? 'admonition_icon_'
+      data[key] = ::Hash[(val || {}).map {|(key2, val2)|
```

Comment:
Personally, I try to avoid using the curly-brace syntax for multi-line blocks.

I'd probably break this out into a local variable something like:
```ruby
inner = (val || {}).map do |key2, val2|
  normalized = evaluate val2, data
  [key2.to_sym, (key2.end_with? '_color') ? to_color(normalized) : normalized]
end

data[key] = ::Hash[inner] 
```

But that's just my opinion, I'd say choose whatever you find easier to maintain :smiley: